Tests/XcodeGenKitTests/ProjectGeneratorTests.swift

```diff
@@ -484,7 +484,33 @@ class ProjectGeneratorTests: XCTestCase {
                 try expect(targetConfig1.buildSettings["ASSETCATALOG_COMPILER_APPICON_NAME"] as? String) == "AppIcon"
                 try expect(targetConfig1.buildSettings["CODE_SIGN_IDENTITY"] as? String) == "iPhone Developer"
             }
-            
+
+            $0.it("supportedDestinations merges settings - iOS, watchOS") {
```

Comment:
These tests can be parameterized tests to reduce redundancy.

But it's better existing test cases also become parameterized tests.
So I think it's good at this time.